arbcompress/compress_common.go

```diff
@@ -3,14 +3,13 @@
 
 package arbcompress
 
-const LEVEL_FAST = 0
 const LEVEL_WELL = 11
 const WINDOW_SIZE = 22 // BROTLI_DEFAULT_WINDOW
 
 func compressedBufferSizeFor(length int) int {
 	return length + (length>>10)*8 + 64 // actual limit is: length + (length >> 14) * 4 + 6
 }
 
-func CompressFast(input []byte) ([]byte, error) {
-	return compressLevel(input, LEVEL_FAST)
+func CompressFast(input []byte, level int) ([]byte, error) {
```

Comment:
Better to expose `CompressLevel(input, level)`.  `CompressFast(input)` should be left alone so it is for fast compression.